react-ui/src/pages/submission/SubmissionForm.jsx

```diff
@@ -357,36 +373,42 @@ function SubmissionForm({
     }
   };
 
-  const disableSubmitting = submitting || isMutating;
+  const nameCharRemaining = CHAR_LIMIT_NAME - formState.name.value.length;
+  const disableEditing = submitting || isMutating;
+  const disableSubmitting = disableEditing;
 
   const classes = useStyles();
 
   return (
-    <ProtectedRoute
-      message="You must log in with Reddit to submit a flag"
-      showCancel={false}
-    >
+    <ProtectedRoute message="You must log in with Reddit to submit a flag" showCancel={false}>
       <form id="submission-form">
-        <Fieldset disabled={disableSubmitting || submissionExpired}>
-          <TextField
-            id="username"
-            variant="filled"
-            label="Username"
-            disabled
-            value={username}
-          />
+        <Fieldset disabled={disableEditing || submissionExpired}>
+          <TextField id="username" variant="filled" label="Username" disabled value={username} />
           <TextField
             id="name"
             name="name"
             color="secondary"
             variant="filled"
-            helperText={formState.name.error || 'A concise name for your flag'}
+            helperText={(
+              <div className={classes.flex}>
+                <span>
+                  {formState.name.error || ''}
+                  A concise name for your flag.
```

Comment:
This was previously built to only show the error message if there was an error instead of along side the description. I think it's best to keep it that way, so the call to action is clearer instead of being next to more text.

Reply:
You sure? If someone missed the title the first time, then they will never see the helper text.
What if I put in a line break after the error message, so the error message is at least first and on it's own line?

Reply:
As soon as they type to fix the error, they should see the helper text. This is the recommended Material Design approach: https://m3.material.io/components/text-fields/guidelines#9ad90554-a793-4506-9075-6812fd7b381a

Reply:
Ah gotcha, I was wondering what was going on with the helper text after typing but didn't have the time to dig into the MUI docs. Reverted back to `formState.name.error || helperText` convention.